test/lib/formCache.test.ts

```diff
@@ -2,44 +2,30 @@ import { v4 as uuidv4 } from "uuid";
 import { FormCacheFactory, IFormCache } from "../../src/lib/formCache";
 import { Registration } from "../../src/lib/registration/registration";
 
-describe("FormCache", () => {
-  let cache: IFormCache;
-  let id: string;
-
-  beforeEach(() => {
-    cache = FormCacheFactory.getCache();
-    id = uuidv4();
-  });
+jest.mock(
+  "redis-json",
+  () =>
+    function () {
+      return {
+        set: jest.fn(),
+        get: jest.fn(),
+        del: jest.fn(),
+      };
+    }
+);
 
+describe("FormCache", () => {
   it("should return a singleton instance of the form cache", () => {
-    const secondInstance: IFormCache = FormCacheFactory.getCache();
-    expect(cache).toBe(secondInstance);
-  });
+    const cache = FormCacheFactory.getCache();
 
-  it("should return an instance of a registration class for an unknown id", () => {
-    expect(cache.get(uuidv4())).toBeInstanceOf(Registration);
-  });
+    const secondInstance: IFormCache = FormCacheFactory.getCache();
 
-  it("should return the same instance after it is updated", () => {
-    const registration = cache.get(id);
-    cache.update(id, { hexId: "Beacon life" });
-    expect(cache.get(id)).toBe(registration);
+    expect(cache).toBe(secondInstance);
   });
```

Comment:
No longer relevant.  `Registration` is a new instance of the same data each time.

(I was uncomfortable with this test, as it is coupled to the implementation of the cache.  Think we should avoid testing "is instance of" in future as a general rule.)